src/pkg/packaging/windows/host/generatemsi.ps1

```diff
@@ -35,7 +35,7 @@ function RunCandle
         -out "$WixObjRoot\" `
         -ext WixDependencyExtension.dll `
         -dHostSrc="$SharedHostPublishRoot" `
-        -dMicrosoftEula="$PackagingRoot\osx\sharedhost\resources\en.lproj\eula.rtf" `
+        -dMicrosoftEula="$PackagingRoot\osx\license\msft\resources\en.lproj\eula.rtf" `
```

Comment:
Does it still make sense for the Windows installer to get the license from `osx`, now that they aren't shared anymore?

Would it make more sense to put the msft license in the `packaging/windows` folder now?